### PR TITLE
Fix build error

### PR DESCRIPTION
**Pull request by fjvallarino**
@Rise-Vision/apps please review

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -24,7 +25,8 @@
     "uri.js": "1.14.1",
     "lunr.js": "0.5.5",
     "angular": "~1.5.0",
-    "angular-bootstrap": "~0.13.0",
+    "bootstrap": "3.3.7",
```
Comment (alex-deaconu) [inline bower.json]: @fjvallarino curious, why is bootstrap needed here? Thanks.

**Comment by alex-deaconu** [comment]
@fjvallarino looks good, one question inline. Also, re: CH update, are you going to do that in a separate PR? Thanks.

File: bower.json (hunk 1 above)
Comment (fjvallarino) [reply bower.json]: @alex-deaconu it was failing when parsing some files in Bootstrap 4.0.0. Not really sure why it was parsing the contents of that dependency

**Comment by fjvallarino** [comment]
@alex-deaconu I can do it here, no problem. I created this PR mainly because there is a separate branch which is failing the build process.

I'm still working on fixing the vulnerabilities part, which could actually be complex (I'm planning on using a separate PR for this one). Unless I can force a version of a dependency, it means I need to upgrade Jekyll to a different major version which has API changes.

**Comment by alex-deaconu** [comment]
@fjvallarino sounds good. I'm good with having everything in separate PRs. 👍 Thanks!

**Pull request merged**